test/e2e-common.sh

```diff
@@ -20,12 +20,34 @@
 E2E_MIN_CLUSTER_NODES=4
 E2E_MAX_CLUSTER_NODES=4
 
+# Change these two knobs to get a different Istio configuration
```

Comment:
Is this change intended to be merged? You won't be able to control these 2 parameters without changing the source code (e.g., from a Prow job, or from command-line).

/hold

Reply:
refactored to flags.